modules/codegen-plugin/src/smithy4s/codegen/CodegenPlugin.scala

```diff
@@ -64,13 +67,15 @@ object Smithy4sCodegenPlugin extends AutoPlugin {
     smithy4sVersion := BuildInfo.version
   )
 
+  override def projectConfigurations: Seq[Configuration] = Seq(Smithy4s)
+
   override lazy val projectSettings =
     Seq(
       Compile / smithy4sInputDir := (Compile / sourceDirectory).value / "smithy",
       Compile / smithy4sOutputDir := (Compile / sourceManaged).value,
       Compile / smithy4sOpenapiDir := (Compile / resourceManaged).value,
       Compile / smithy4sCodegen := cachedSmithyCodegen(Compile).value,
-      Compile / smithy4sCodegenDependencies := List.empty,
+      Compile / smithy4sCodegenDependencies := Nil: @annotation.nowarn,
```

Comment:
For my own education, what does using `Nil` rather than `List.empty` get us here?

Reply:
absolutely nothing, I'll align with the line below